cmd/experimental/kueue-viz/README.md

```diff
@@ -0,0 +1,62 @@
+# Build and run on OpenShift
```

Comment:
Kueue aims to be vendor neutral. Can you update the main README to show how to build it and run on kind.
For Openshift I would suggest to host another project, or just a small section, not the main entry point for docs.

If this is more work I'm ok in a follow up.

Reply:
done